src/test/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/legalrepresentative/LegalRepresentativeRemissionDecisionPartiallyApprovedPersonalisationTest.java

```diff
@@ -0,0 +1,74 @@
+class LegalRepresentativeRemissionDecisionPartiallyApprovedPersonalisationTest {
+
+    @Mock private AsylumCase asylumCase;
+
+    private String appealReferenceNumber = "someReferenceNumber";
+    private String legalRepRefNumber = "someLegalRepRefNumber";
+    private String appellantGivenNames = "someAppellantGivenNames";
+    private String appellantFamilyName = "someAppellantFamilyName";
+    private String iaExUiFrontendUrl = "http://somefrontendurl";
+    private String templateId = "someTemplateId";
+    private String legalRepEmailAddress = "legalRepEmailAddress@example.com";
+    private String amountLeftToPay = "40.00";
+
+    private LegalRepresentativeRemissionDecisionPartiallyApprovedPersonalisation legalRepresentativeRemissionDecisionPartiallyApprovedPersonalisation;
+
+    @BeforeEach
+    void setUp() {
+
+        legalRepresentativeRemissionDecisionPartiallyApprovedPersonalisation =
+            new LegalRepresentativeRemissionDecisionPartiallyApprovedPersonalisation(templateId, iaExUiFrontendUrl);
+    }
+
+    @Test
+    void should_return_given_template_id() {
+        assertEquals(templateId, legalRepresentativeRemissionDecisionPartiallyApprovedPersonalisation.getTemplateId(asylumCase));
+    }
+
+    @Test
+    void should_return_given_email_address_from_asylum_case() {
+        when(asylumCase.read(LEGAL_REPRESENTATIVE_EMAIL_ADDRESS, String.class))
+            .thenReturn(Optional.of(legalRepEmailAddress));
+
+        assertTrue(
+            legalRepresentativeRemissionDecisionPartiallyApprovedPersonalisation.getRecipientsList(asylumCase).contains(legalRepEmailAddress));
+    }
+
+    @Test
+    void should_return_personalisation_when_all_information_given() {
+
+        when(asylumCase.read(APPEAL_REFERENCE_NUMBER, String.class)).thenReturn(Optional.of(appealReferenceNumber));
+
+        when(asylumCase.read(LEGAL_REP_REFERENCE_NUMBER, String.class)).thenReturn(Optional.of(legalRepRefNumber));
+        when(asylumCase.read(APPELLANT_GIVEN_NAMES, String.class)).thenReturn(Optional.of(appellantGivenNames));
+        when(asylumCase.read(APPELLANT_FAMILY_NAME, String.class)).thenReturn(Optional.of(appellantFamilyName));
+        when(asylumCase.read(AMOUNT_LEFT_TO_PAY, String.class)).thenReturn(Optional.of(amountLeftToPay));
+
+        Map<String, String> personalisation =
+            legalRepresentativeRemissionDecisionPartiallyApprovedPersonalisation.getPersonalisation(asylumCase);
+
+        assertEquals(appealReferenceNumber, personalisation.get("appealReferenceNumber"));
+        assertEquals(legalRepRefNumber, personalisation.get("legalRepReferenceNumber"));
+        assertEquals(appellantGivenNames, personalisation.get("appellantGivenNames"));
+        assertEquals(appellantFamilyName, personalisation.get("appellantFamilyName"));
+        assertEquals(iaExUiFrontendUrl, personalisation.get("linkToOnlineService"));
+        assertEquals(amountLeftToPay, personalisation.get("feeAmount"));
+    }
```

Comment:
You will need to test for customer services telephone and email address.

Reply:
Added the telephoneNumber and emailAddress. Modified the template in Gov Notify